utils/validation.py

```diff
@@ -161,3 +161,7 @@ def validate_ping(ping, ping_pool=None, vote=False):
         return False
 
     return True
+
+# TODO: Implement validation for an entire chain
```

Comment:
make sure that the implementation of this reuses validate_tick() (which in turn uses validate_ping()) that way we have max reusability and minimum code duplication